composer.json

```diff
@@ -20,6 +20,6 @@
         "classmap": ["src/geshi/", "src/geshi.php"]
     },
     "require-dev": {
-        "phpunit/phpunit": "^5.7"
+        "phpunit/phpunit": "^5.7 || ^6.5 || ^7.5 || ^8.2"
```

Comment:
Don't quite like having to depend on at least 4 versions half of which are out of support, but well, compat issue in phpunit.

Reply:
Different phpunit versions are needed for different PHP versions. No way around it.

Reply:
Sure. As said: Don't quite like it, but can't be avoided.

Reply:
A `*` might work. It would always pick the newest phpunit available for the available php version.

Reply:
Sound's like a route to go.